api/python/cell_census/src/cell_census/util.py

```diff
@@ -13,3 +16,42 @@ def uri_join(base: str, url: str) -> str:
     path = urllib.parse.urljoin(p_base.path, p_url.path)
     parts = [p_base.scheme, p_base.netloc, path, p_url.params, p_url.query, p_url.fragment]
     return urllib.parse.urlunparse(parts)
+
+
+def get_experiment(census: soma.Collection, organism: str) -> soma.Experiment:
```

Comment:
The `util.py` package is probably best for methods that are not census-specific. Seems like this could be put into a new `get_experiment.py` package, analogous to `get_presence_matrix.py`.  Though I might also remove `get_` from these package names.

Reply:
done